samples/error-correction/syndrome/Syndrome.qs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+namespace Microsoft.Quantum.Samples.Hardware.Syndrome
+{
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Preparation;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Core;
+
+    /// # Summary
+    /// Apply a controlled Pauli operation for a given basis and control/target qubits
+    ///
+    /// # Input
+    /// ## basis
+    /// Pauli basis in which to apply the controlled pauli
+    /// ## control
+    /// Control qubit
+    /// ## target
+    /// Target qubit
+    operation ControlledPauli(basis: Pauli, control: Qubit, target: Qubit): Unit {
+        if (PauliZ == basis) {
+            CZ(control, target);
+        }
+        if (PauliX == basis) {
+            CX(control, target);
+        }
+        if (PauliY == basis) {
+            CY(control, target);
+        }
+    }
+
+    /// # Summary
+    /// Measure qubit in a given basis and return the result
+    ///
+    /// # Input
+    /// ## basis_qubit
+    /// Tuple of the Pauli basis and qubit to measure
+    /// 
+    /// # Output
+    /// ## result
+    /// Measurement result
+    operation BasisMeasure(basis_qubit: (Pauli, Qubit)): Result {
+        let (basis, qubit) = basis_qubit;
+        let result = Measure([basis], [qubit]);
+        Reset(qubit);
+        return result;
+    }
+
+    operation Prepare(qubit: Qubit, value: Bool, basis: Pauli): Unit {
```

Comment:
I think the body is fairly self-explanatory, but an API documentation comment may help new readers nonetheless; I could see it going either way in this case.

Reply:
Sorry, I missed documenting this one.

Reply:
By the way, this may also be a function we want to add to the standard libraries. What do you think?